atd-vzv/src/Components/Popover/popoverConfig.js

```diff
@@ -18,19 +22,68 @@ export const popoverConfig = {
         </>
       ),
     },
-    raceEthnicity: {
-      title: "Race/Ethnicity",
+    demographics: (isMobile) => ({
+      title: "Demographics Data",
       html: (
         <>
+          <div className="font-weight-bold">Demographics Data</div>
+          <div className="mb-2">
+            To compare crash demographics data with historical and estimated
+            demographics data for the City of Austin geographic boundaries,
+            visit the US Census Bureau's latest{" "}
+            <a
+              href="https://data.census.gov/cedsci/table?g=1600000US4805000&tid=ACSDP5Y2018.DP05&t=Hispanic%20or%20Latino&layer=VT_2018_160_00_PY_D1&hidePreview=true&moe=false"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              demographics estimates
+            </a>
+            .
+          </div>
+          <div className="font-weight-bold">Gender</div>
+          <div className="mb-2">
+            This visualization relies on the information entered by officers
+            into the official crash reports. Officers are encouraged to use the
+            gender that appears on the Driver License/ID Card.
+          </div>
+          <div className="font-weight-bold">Gender</div>
+          <div className="mb-2">
+            This visualization relies on the information entered by officers
+            into the official crash reports. Officers are encouraged to use the
+            gender that appears on the Driver License/ID Card.
+          </div>
```

Comment:
It looks like you've got Gender showing twice:

![2020-07-12_21-42-19](https://user-images.githubusercontent.com/5282430/87265492-91447300-c488-11ea-825f-dac7c6c24733.png)

Reply:
Thanks for catching this, and thanks @mateoclarke for fixing it up before merge. 🙏